### PR TITLE
Feature/refactor plugins for require

### PR DESCRIPTION
**Pull request by ajmcnall**
Resolves ceros/ceros-product#2688

for your approval

### CODE REVIEW

File: src/soundjs/soundjs.js [hunk 1]
```diff
@@ -19,29 +19,22 @@
...
-    function activatePlaySound() {
-        loadSoundJs();
+        paths: { 
+            CerosSDK: "//sdk.ceros.com/standalone-player-sdk-v3",        
+            SoundJS: "https://code.createjs.com/soundjs-0.6.2.min",
+        }
+        
+    });
 
+    require(['CerosSDK', 'SoundJS'], function (CerosSDK, createjs) {
```
Comment (danwashable) [inline src/soundjs/soundjs.js]: Weird that the SoundJS variable is called `createjs` here. Since this can be anything, `SoundJS` would be more accurate.

**Comment by danwashable** [comment]
@ajmcnall @tommydarko Just one comment on the code but it's on the sound plugin which I thought wasn't supposed to be in here anymore. Does the sound plugin still need to be taken out of this PR?

**Comment by tommydarko** [comment]
@danwashable as, in theory, there could be people using it from this repo's URL I would suggest we keep it here for the time being. Going forward we'll be using the CerosLabs repo for all new none-core plugins.

**Comment by danwashable** [comment]
@tommydarko OK, sounds good! Then there's one small code item to address and then we can move on to testing.

File: src/soundjs/soundjs.js (hunk 1 above)
Comment (tommydarko) [inline src/soundjs/soundjs.js]: This might me a RequireJS hack as SoundJS isn't really AMD... let me confirm that and either fix or document it.

**Comment by tommydarko** [comment]
Superseded by https://github.com/ceros/ceros-plugins/pull/17

**Pull request closed without merging**